Sources/FoundationEssentials/ProcessInfo/ProcessInfo.swift

```diff
@@ -276,8 +424,20 @@ extension _ProcessInfo {
             return 0
         }
         return Int(count)
-#else
+#elseif os(Linux) || os(FreeBSD)
         return Int(sysconf(Int32(_SC_NPROCESSORS_ONLN)))
+#elseif os(Windows)
+        var sysInfo = SYSTEM_INFO()
+        GetSystemInfo(&sysInfo)
+        let activeProcessorMask = sysInfo.dwActiveProcessorMask
+        // assumes sizeof(DWORD_PTR) is 64 bits or less
+        var v = activeProcessorMask
+        v = v - ((v >> 1) & 0x5555555555555555)
+        v = (v & 0x3333333333333333) + ((v >> 2) & 0x3333333333333333)
+        v = (v + (v >> 4)) & 0xf0f0f0f0f0f0f0f
+        return (v * 0x0101010101010101) >> ((sizeof(v) - 1) * 8)
```

Comment:
This is just popcount isn't it? We are in swift, so why not just do `sysInfo.dwActiveProcessMask.nonzeroBitCount`? That should lower to popcount.

Reply:
It is! Good call, updated 👍🏻